src/extensions/extensionInterface.ts

```diff
@@ -333,6 +343,29 @@ export const recordingSelectionReducer: Reducer<RecordingSelection, RecordingSel
             timeRange: action.timeRange
         }
     }
+    else if (action.type === 'ZoomTimeRange') {
+        const maxTimeSpan = 30000 * 60 * 5
+        const currentTimepoint = state.currentTimepoint
+        const timeRange = state.timeRange
+        if (!timeRange) return state
+        if ((timeRange.max - timeRange.min) / action.factor > maxTimeSpan ) return state
```

Comment:
not just reduce to the max allowed time span?

Reply:
TODO: reduce to the max allowed time span in this case